### ISSUE TITLE
Screen reader a11y: PublicationListElement wrong labels

### ISSUE THREAD

**Issue by danielweck** [readium/readium-desktop#600]
The ARIA labels are announced for each paragraph, but this erases the actual values in each paragraph. ARIA labelled-by is probably more semantically correct, although this would require 

https://github.com/readium/readium-desktop/blob/41733d4b5ece3d1810f2972ae107c7b811c1c920/src/renderer/components/publication/PublicationListElement.tsx#L100

https://github.com/readium/readium-desktop/blob/41733d4b5ece3d1810f2972ae107c7b811c1c920/src/renderer/components/publication/PublicationListElement.tsx#L108

https://github.com/readium/readium-desktop/blob/41733d4b5ece3d1810f2972ae107c7b811c1c920/src/renderer/components/publication/PublicationListElement.tsx#L110

https://github.com/readium/readium-desktop/blob/41733d4b5ece3d1810f2972ae107c7b811c1c920/src/renderer/components/publication/PublicationListElement.tsx#L103

**Comment by danielweck** [comment]
...which brings another issue: I am not sure there should be translated labels specifically for screen reader users. For example, if "book title" is displayed nowhere in the UI for sighted users, then there is no reason why non-sighted users would need to hear this. Conversely, ARIA labelled-by would typically be used for both sighted and VI users, for UI forms / input controls.

https://github.com/readium/readium-desktop/blob/41733d4b5ece3d1810f2972ae107c7b811c1c920/src/resources/locales/en.json#L2-L12

**Comment by danielweck** [comment]
I filed a separate issue for misuse of a11y labelling:  https://github.com/readium/readium-desktop/issues/601

**Comment by danielweck** [comment]
Also note that ARIA labels inside `aria-hidden` fragment is non-sensical:

https://github.com/readium/readium-desktop/blob/1263712b0ddab6ad43e3c95f45a9ed74c831960d/src/renderer/components/publication/PublicationCard.tsx#L86-L93

**Comment by Scarsniik** [comment]
Fixes by #631
